ckanext/dcat/profiles.py

```diff
@@ -1223,8 +1239,12 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):
         self._add_triples_from_dict(dataset_dict, dataset_ref, items)
 
         # Tags
-        for tag in dataset_dict.get('tags', []):
-            g.add((dataset_ref, DCAT.keyword, Literal(tag['name'])))
+        for tag in dataset_dict.get('tags_translated', dataset_dict.get('tags', [])):
```

Comment:
@wardi In CKAN Core, are the translated fields just stored like this:
```
{
  'title': {
    'en': 'English Title',
    'fr': 'French Title',
  }
}
```

and then in fluent, they are stored like:
```
{
  'title_translated': {
    'en': 'English Title',
    'fr': 'French Title',
  }
}
```

The `tags_translated` is a Fluent thing as they are partially supported in fluent. They are just not really used/fully supported everywhere where the Core tag features are.

Reply:
`fluent_tags` *is* what we're using to implement our translated keywords field. Because they're represented differently we store them as `keywords` and not `tags_translated`. I don't know how many other sites are using this fluent feature and how they store them in their schemas, but at least for our schema this won't work, right?

Reply:
Ah okay I understand now. Umm I just added this feature to this PR. But it is the same one that I have going for the core PR.

So I can change this field key to what our keywords field key is after we merge this, if that is fine?